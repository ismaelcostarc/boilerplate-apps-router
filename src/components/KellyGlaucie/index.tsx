import * as S from "./styles"; const
KellyGlaucie
= () => { return (
<S.Wrapper>
  <h1>KellyGlaucie</h1>
</S.Wrapper>
); }; export default
KellyGlaucie;