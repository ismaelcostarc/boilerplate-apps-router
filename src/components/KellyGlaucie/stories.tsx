import { Meta, StoryObj } from "@storybook/react";
import KellyGlaucie from ".";
export default {
  title: "KellyGlaucie",
  component: KellyGlaucie,
  parameters: { layout: "fullscreen" },
} as Meta;
export const Default: StoryObj = {};
