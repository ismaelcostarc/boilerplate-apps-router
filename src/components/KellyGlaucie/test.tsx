import { render, screen } from "@testing-library/react";
import KellyGlaucie from ".";

describe("KellyGlaucie", () => {
  it("should render the component", () => {
    const { container } = render(<KellyGlaucie />);

    expect(screen.getByRole("heading", { name: /KellyGlaucie/i })).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
