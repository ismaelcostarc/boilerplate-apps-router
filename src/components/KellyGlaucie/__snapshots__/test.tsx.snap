// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KellyGlaucie should render the component 1`] = `
<div
  class=""
>
  <h1>
    KellyGlaucie
  </h1>
</div>
`;
